Controllers/feedbackController.js

```diff
@@ -17,6 +18,19 @@ const logger = require('../utility/logger/logger');
 feedback forms.
  */
 
+const CACHE_TTL = 300; // cache time to live: 5 minutes
+
+const getCachedFeedBack = async (key) => {
```

Comment:
Since this is a function that get's used in the controller, let's move it to the utility folder and import it here for use.
similar to how the input validator is used.